ethcore/src/engines/mod.rs

```diff
@@ -71,8 +71,9 @@ pub trait Engine : Sync + Send {
 	/// Block transformation functions, after the transactions.
 	fn on_close_block(&self, _block: &mut ExecutedBlock) {}
 
-	/// If true, generate_seal has to be implemented.
-	fn seals_internally(&self) -> bool { false }
+	/// If Some(true) this author is able to generate seals, generate_seal has to be implemented.
```

Comment:
Perhaps a 3-value `enum` would indicate those intentions more? It was my first thought - why is it an `Option`?